release-notes/6.0/preview/api-diff/preview1/Asp.Net/6.0-preview1.md

```diff
@@ -0,0 +1,27 @@
+# API Difference 5.0.0 vs 6.0.0-preview1
+
+API listing follows standard diff formatting. Lines preceded by a '+' are
+additions and a '-' indicates removal.
```

Comment:
Not positive how best to phrase it but I suggest adding a link to aspnet/Announcements#444 in this doc or release-notes/6.0/preview/api-diff/preview1/README.md. The nullability changes may impact our users _more_ than (say) the parameter name changes documented here.